docs/sources/panels/visualizations/table-panel.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Table panel"
+keywords = ["grafana", "dashboard", "documentation", "panels", "table panel"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/table/", "/docs/grafana/latest/features/panels/table_panel/"]
+[menu.docs]
+name = "Table panel"
+parent = "visualizations"
+weight = 500
+draft = "true"
++++
+
+
+# Table panel
+
+The table panel is very flexible, supporting multiple modes for time series and for
+tables, annotation, and raw JSON data. This panel also provides date formatting, value formatting, and coloring options.
+
+<img class="screenshot" src="/assets/img/features/table-panel.png">
+
+To view table panels in action and test different configurations with sample data, check out the [Table Panel Showcase in the Grafana Playground](https://play.grafana.org/dashboard/db/table-panel-showcase).
```

Comment:
This link is no longer relevant, we need to prepare a now showcase dashboard for the new Table panel (https://github.com/grafana/grafana/issues/23904)

Reply:
Removed link, added placeholder. Also put a note in the linked issue to update this. 

Since the topic is marked as draft, a broken link is not a blocker for this PR.